bin/ci

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+function usage_ci() {
+    echo -e "Usage: ${1} [OPTIONS] COMMAND
+
+OPTIONS:
+  -h, --help   print this message
+
+COMMAND:
+  check_configuration  check that the \".circleci/config.yml\" file is up-to-date
+  update               generate the \".circleci/config.yml\" file for
+                       the current diff as compared to the master branch
+"
+}
+
+
+function check_changes() {
+    git whatchanged --name-only --pretty="" master | grep "${1}" &> /dev/null
+}
+
+
+# Utility function that updates circle configuration source files with a
+# workflow for each site and running only the necessary jobs
+#
+# Usage: update_sources
+function update_sources() {
+    # Reset the content of the .circleci/src/@root.yml file
+    disclaimer="# /!\ DO NOT EDIT: this file is autogenerated"
+    printf "%s\nparameters:\n" "${disclaimer}" > ".circleci/src/@parameters.yml"
+
+    while IFS= read -r -d '' site_path
+    do
+        # shellcheck disable=SC2001
+        site=$(echo "${site_path}" | sed 's|'"${SITES_DIRECTORY}\/"'||')
```

Comment:
Use parameter substitution instead, `shellcheck` will be happy with this:

```suggestion
        site=$(echo "${site_path//${SITES_DIRECTORY}\//}")
```

Reply:
It works!